clients/spark/core/src/main/scala/io/treeverse/clients/GCBackupAndRestore.scala

```diff
@@ -78,16 +82,52 @@ object GCBackupAndRestore {
     }
   }
 
+
+
+  // control copy speed
+  //  public static final String CONF_LABEL_DIRECT_WRITE = "distcp.direct.write";
```

Comment:
I'd prefer to have no code in comments.  But if you _really_ want it, could you please expand the comment on line 87?